src/FileProviders/Embedded/src/Manifest/ManifestParser.cs

```diff
@@ -146,7 +146,7 @@ private static string EnsureText(XElement element)
             if (element.Elements().Count() == 0 &&
                 !element.IsEmpty &&
```

Comment:
❔ Should `IsEmpty` also have this?

```
[MemberNotNullWhen(false, nameof(FirstNode))]
```

Reply:
Perhaps. It is from XElement in runtime.